modules/browsiRtdProvider.js

```diff
@@ -33,14 +36,20 @@ let _dataReadyCallback = null;
 
 /**
  * add browsi script to page
- * @param {string} bptUrl
+ * @param {Object} data
  */
-export function addBrowsiTag(bptUrl) {
-  let script = loadExternalScript(bptUrl, 'browsi');
+export function addBrowsiTag(data) {
+  let script = loadExternalScript(data.u, 'browsi');
+  script.async = true;
   script.setAttribute('data-sitekey', _moduleParams.siteKey);
   script.setAttribute('data-pubkey', _moduleParams.pubKey);
   script.setAttribute('prebidbpt', 'true');
   script.setAttribute('id', 'browsi-tag');
+  script.setAttribute('src', data.u);
+  script.prebidData = utils.deepClone(data);
+  if (_moduleParams.keyName) {
+    script.prebidData.kn = _moduleParams.keyName;
+  }
```

Comment:
Just to confirm - this specific setup with for the `prebidData` property is correct?  

I tried to load the `browsi_test_Page.html` and I couldn't really confirm if this key and its values were being properly used by the script; so I wanted to ask and confirm in case that didn't sound right to you.

Reply:
Yes, it's correct
I updated the demo page so you can see this there as well